writer/tests/system/write/test_occ_locking.py

```diff
@@ -34,6 +34,10 @@ def create_and_update_model(json_client, fqid, create_payload, update_payload):
         },
     )
     assert response.status_code == 200
+    update_model(json_client, fqid, update_payload)
```

Comment:
Is this split necessary?

Reply:
Nope, thought I needed update_model seperately. Reverted